plugins/gopacket/main.go

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2021 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"github.com/urfave/cli"
+)
+
+func main() {
+	viper.AutomaticEnv()
+
+	app := cli.NewApp()
+	app.Name = "APIClarity packet source extension"
+	app.Version = "0.1.0"
```

Comment:
we need to set it from the build version

Reply:
You mean to import backend go.mod

Reply:
no, I mean to add a version pkg and set the values during docker build time like we are doing in the backend build.